src/pages/FeedbackModal.jsx

```diff
@@ -12,12 +14,23 @@ export const FeedbackModal = ({ visible, setVisible }) => {
 
   const [ratingValue, setRatingValue] = useState(0)
   const [reviewText, setReviewText] = useState('')
+  const [isFeedbackEditModalVisible, setFeedbackEditModalVisible] = useState(false)
+  const [editReview, setReview] = useState('')
+
+  const handleEditFeedbackClick = (id) => {
+    const review = reviews.find((r) => r.id === id)
```

Comment:
find function can return `null`, so better to check, if you have review then open edit modal